### DISCUSSION TITLE
Problem with pi zero 2w

### DISCUSSION THREAD

**Discussion by maurensen** [frntc/RAD#33] [Q&A]
Hi, as title suggests I have problem with rad and a pi zero 2w. The rad is working ok with a pi3 but when connecting a zero 2 I only get the c64 basic screen when the rad is powered. I always tried tweaking some values as suggested on rad.cfg but without success. Any suggestion would be appreciated. Thanks on advance!

**Comment by futureretrofusion** [comment]
Does the  Rad try to reset the c64 at boot, before going back to basic?

Have you tried with just empty firmware?

My pi zero 2w just work with the same settings as a pi3.

**Comment by maurensen** [comment]
No, the 64 boots like there's no rad connected. Yes it's very strange because I'm using the very same as card working on 3b.... For the record it's a c64c pal latest version with no screws

**Comment by futureretrofusion** [comment]
Have you tried booting anything else on the 2w on its own? Hooked up to the
hdmi to ensure the pi is working?

Any led activity on the pi?

**Reply by maurensen** [reply]
Sadly the pi zero 2 has only a green led that's always on no blinking when accessing the SD card like the pi 3. Btw damn myself I've forgotten to say that I've successfully booted 1 time the rad with the zero but I was able to run only prg files nuvies were not working. So I tried to modify the Rad.cfg and now nothing is working now. I've tried to restart from the beginning with a blank installation but I'm not able to boot the rad with zero anymore...  I'm a f....g idiot ;-(

**Comment by futureretrofusion** [comment]
So it is just an issue with the pi zero? Have you tried just booting Linux
on it? Could have it become faulty?

These pizero2w have become cheap I can get them for around $15us new these
days.
